src/r_fps.c

```diff
@@ -207,21 +209,27 @@ void R_UpdateViewInterpolation(void)
 
 		if (oldview_invalid[i]) oldview_invalid[i]--;
 	}
+
+	last_view_update = I_GetTime();
 }
 
 void R_ResetViewInterpolation(UINT8 p)
 {
+	// Wait an extra tic if the interpolation state hasn't
+	// updated yet.
+	int t = last_view_update == I_GetTime() ? 1 : 2;
+
 	if (p == 0)
 	{
 		UINT8 i;
 		for (i = 0; i < MAXSPLITSCREENPLAYERS; i++)
 		{
-			oldview_invalid[i]++;
+			oldview_invalid[i] = t;
```

Comment:
Shouldn't that be `+=` instead of `=`?

Reply:
well lol https://github.com/KartKrewDev/RingRacers/blob/dd45f3cd61a6afa83cac876a0ae7d43fa2d1ab43/src/r_fps.c#L175

Reply:
ye uh lmao its intentional so `"do not add extra time for subsequent calls"`